src/audio/volume/volume_ipc4.c

```diff
@@ -94,16 +94,8 @@ static void init_ramp(struct vol_data *cd, uint32_t curve_duration, uint32_t tar
 						  Q_CONVERT_FLOAT(1.0 / 10000, 31), 0, 31, 0);
 	}
 
-	if (!cd->initial_ramp) {
-		/* In case when initial ramp time is equal to zero, vol_min and
-		 * vol_max variables should be set to target_volume value
-		 */
-		cd->vol_min = target_volume;
-		cd->vol_max = target_volume;
-	} else {
-		cd->vol_min = VOL_MIN;
-		cd->vol_max = VOL_MAX;
-	}
+	cd->vol_min = VOL_MIN;
+	cd->vol_max = VOL_MAX;
```

Comment:
@RanderWang can you review, this is removing the logic in commit 751d0a967e765ac193e7e75b38c30b11bbc37c51

Reply:
sure.